src/components/core/Bookmark/BookmarkIcon.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { func } from "prop-types";
+
+import { BookmarkHollowIcon } from "assets/icons/BookmarkHollow";
+import { BookmarkFilledIcon } from "assets/icons/BookmarkFilled";
+
+const Container = styled.div`
+  cursor: pointer;
+  display: flex;
+  margin-left: 20px;
+  height: 40px;
+  width: 40px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 50%;
+  :hover {
+    background-color: #f6f6f7;
+  }
+`;
+
+const BookmarkIcon = ({ isBookmarked, handleClick }) => {
+  return (
+    <Container onClick={handleClick}>
```

Comment:
I'm not sure about this component from the semantic side, as div shouldn't be used as a button